File: HackerRank - Time Conversion/solver.py
#!/bin/python3

import math
import os
import random
import re
import sys

#
# Complete the 'timeConversion' function below.
#
# The function is expected to return a STRING.
# The function accepts STRING s as parameter.
#


def timeConversion(s):
    # Write your code here
    time = s.split(":")
    h = time[0]
    if "PM" in s:
        if h != "12":
            time[0] = str(int(h) + 12)
    else:
        if h == "12":
            time[0] = "00"
    s = ":".join(time)
    return s[:-2]


if __name__ == '__main__':
    fptr = open(os.environ['OUTPUT_PATH'], 'w')

    s = input()

    result = timeConversion(s)

    fptr.write(result + '\n')

    fptr.close()

File: Paragon - Hiring Test/Problem4/solver.ts
function mengintip_kucing(n: number, tinggi_kucing: number[]): number {
    // Your code starts here.
    let result: number = 0;

    for (let i = 0; i < n; i++) {
        for (let j = i - 1; j >= 0; j--) {
            if (tinggi_kucing[j] <= tinggi_kucing[i]) {
                result++;
            } else {
                break;
            }
        }
        result++;
    }

    return result;
}

export default mengintip_kucing;
